client/src/annoMatrix/loader.ts

```diff
@@ -355,20 +373,28 @@ function _obsOrVarLoader(baseURL, field, query) {
   return () => doBinaryRequest(url);
 }
 
-// @ts-expect-error ts-migrate(7006) FIXME: Parameter 'baseURL' implicitly has an 'any' type.
-function _XLoader(baseURL, field, query) {
+function _XLoader(
+  baseURL: string,
+  // @ts-expect-error ts-migrate --- TODO revisit:
+  // Compiler is complaining that field is declared but never read.
```

Comment:
AFAIK, the proper fix is just to prefix the parameter name with an underscore, eg, `_field`

Reply:
So easy, thanks Bruce! 

Updated [loader.ts::_XLoader](https://github.com/chanzuckerberg/cellxgene/blob/mim/annoMatrix/client/src/annoMatrix/loader.ts#L371)